src/lib/viewers/archive/ArchiveViewer.js

```diff
@@ -0,0 +1,310 @@
+/* eslint-disable no-unused-vars */
+import './Archive.scss';
+import BaseViewer from '../BaseViewer';
+import { VIEWER_EVENT } from '../../events';
+
+const JS = ['archive.js'];
+const CSS = ['archive.css'];
+
+const testData = [
```

Comment:
Make sure this doesn't get commited

Reply:
I put it there in case you guys want to see how it looks like. Will remove it later.